### DISCUSSION TITLE
How to transfer balance from pre-mined wallets to others?

### DISCUSSION THREAD

**Discussion by jacobhjkim** [0xPolygon/polygon-edge#555] [Q&A] [answered]
When creating the genesis file, I pre-mined with the following command.
```
$ polygon-edge genesis
    ...
    --premine 0x1010101010101010101010101010101010101010:1000000000000000000000
```
I can check that the wallet `0x1010101010101010101010101010101010101010` contains 10^21 Wei ETHs. How do I send this token to a new wallet? Where can I get the private key for the pre-mined wallet? Is it generated with the genesis? (I feel like I am misunderstanding something big about how a wallet works.)

**Comment by 0xAleksaOpacic** [comment, marked as answer]
Hi @jacobhjkim 
When creating the network and the genesis file you should pre-mine only accounts for which you have private keys. If you pre-mine accounts and do not have a private key to access them, your pre-mined balance will not be usable

Solution:
1. You can pre-mine nodes with addresses from the first step (https://edge-docs.polygon.technology/docs/get-started/set-up-ibft-locally#step-1-initialize-data-folders-for-ibft-and-generate-validator-keys). The private key from a node is places in the data-dir of the node, example:
    test-chain-N -> consensus -> validator.key

2. You can use any address from your wallet (for example metamask). Private keys and addresses remain the same when you switch the networks.

**Reply by jacobhjkim** [reply]
Thanks!
